src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -375,6 +375,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('autoReveal', "Controls whether the explorer should automatically reveal and select files when opening them."),
 			'default': true
 		},
+		'explorer.autoRevealScroll': {
+			'type': 'boolean',
+			'description': nls.localize('autoRevealScroll', "Controls whether the explorer should automatically scroll to revealed files."),
```

Comment:
Perhaps -
> Controls whether the explorer should automatically scroll when a different file is selected

Reply:
autoRevealScroll depends on autoReveal being active, so I feel the current description fits better.